binaryaudit/abicheck.py

```diff
@@ -37,8 +37,15 @@ def serialize(fn):
     return process.returncode, out, cmd
 
 
-def compare(ref, cur):
+def compare(ref, cur, suppr=[]):
     cmd = ["abidiff", ref, cur]
+
+    if len(suppr) > 0:
```

Comment:
No need for the condition here. If the array is empty, no iteration will happen. If the condition is to stay, it only has to concern the iteration code, the previously resulting `cmd` should stay same.

Thanks